terminology.rst

```diff
@@ -206,3 +212,17 @@ Raiden Terminology
 
    expiration
         Specific block after which the lock in the :term:`locked transfer` expires
+
+   offchain message
+        A message that is only sent between Raiden nodes, with no information that could be
+        forwarded to a contract - for example, a ``Processed`` message.
+
+   onchain message
```

Comment:
I'm still afraid people might be misled by that name. What do you think of "on-chain capable message"? Not great either, but at least somewhat clearer. I'd like someone else to chime in on this before merging it.

Reply:
IMO, there's no such thing as an on-chain message. I would suggest removing the whole off-chain / on-chain message.

All messages are off-chain. Transactions are executed onchain using certain signatures / data exchanged via messages.

>         A message that is sent between Raiden nodes but could as well be forwarded to a contract.

This is incorrect.

> Notice that a message does not have to be published on a contract to be considered an
        onchain message

Doesn't make much sense really. 

> It only has to be of a type that allows sending it to a contract in total, e. g. ``WithdrawRequest``.

hmm what type is that? 

As i said, we don't need this definition as trying to fit the word `onchain` and `message` is invalid.

Reply:
There are certain sets of messages that are signed in different ways for good reasons. It will be easier to talk about the differences when we give them some name or description, even if it's something long like "Messages that contain a signature that is necessary to create a specific on-chain transaction".

At least for the signatures, there are three different groups of messages. If we can't say anything about these groups of messages in general without getting inaccurate, we should at least write up a proper description on how the signatures work and why they are constructed in the way they are.